### ISSUE TITLE
Create python wheels for package

### ISSUE THREAD

**Issue by SushiWaUmai** [craffel/pretty-midi#211]
Since there doesn't seem to be any wheels on this package, it would be nice you we have this in here, since it would shorten the installation and therefore increase the developer experience.

**Comment by zahlman** [comment]
I'd like to add my voice to this. Nothing in `setup.py` suggests the need for a client-side build step. Creating a wheel for pure-Python projects is easy (and still works with modern tooling even without a `pyproject.toml`; although the latter is still cleaner on the developer's end, as it makes metadata properly declarative and avoids issues with choosing a version of Setuptools or whatever alternative) and improves installation performance. It also helps raise the security baseline of the Python ecosystem: since wheels don't require arbitrary code to run at install time, the install environment can safely be inspected before using the installed packages. (Showing source on GitHub does not assuage the paranoid, as PyPI cannot establish a relationship between the repo and the sdist.)

Ref.: https://pradyunsg.me/blog/2022/12/31/wheels-are-faster-pure-python/ (by core Python developer Pradyun Gedam)

I have workflows that invoke Pip with `--only-binary=:all:` routinely (and others may do the equivalent with uv), which often works in 2026 but is blocked by any sdist-only transitive dependency; so I've been trying to encourage wheels where I encounter the issue and there are no obvious blockers.

**Comment by craffel** [comment]
Ok, I made a new release on PyPI (0.2.11a0) which includes wheels. The code has not changed since then so the only reason for the new release is the wheels (and PyPI won't allow uploading new files to a release > 14 days old anymore, hence the need for a new version identifier). Let me know if you face any issues.

https://pypi.org/project/pretty-midi/0.2.11a0/

**Comment by zahlman** [comment]
Right; that's a relatively new restriction that I forgot about, sorry.

An `a` suffix on a version marks the package as a "pre-release" version, older than 0.2.11 rather than newer. I think you want `0.2.11.post0`. Ref.: https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers

**Comment by craffel** [comment]
Ah good call. I deleted 0.2.11a0 and added 0.2.11.post0. Let me know if there is any issue.

**Comment by zahlman** [comment]
Package resolution now succeeds with `--only-binary=:all:` in a new environment for my use case, thank you.
